docs/main/quickstart-guide/installation.md

```diff
@@ -53,19 +52,19 @@ helm repo add openebs https://openebs.github.io/openebs
 helm repo update
 ```
 
-OpenEBS provides several options that you can customize during install like:
+OpenEBS provides several options that you can customize during installation like:
 - specifying the directory where hostpath volume data is stored or
 - specifying the nodes on which OpenEBS components should be deployed and so forth. 
 
-The default OpenEBS helm chart will install both Local Storage and Replicated Storage. Refer to [OpenEBS helm chart documentation](https://github.com/openebs/charts/tree/master/charts/openebs) for full list of customizable options and using other flavors of OpenEBS data engines by setting the correct helm values. 
+The default OpenEBS helm chart will install both Local Storage and Replicated Storage. Refer to [OpenEBS helm chart documentation](https://github.com/openebs/charts/tree/master/charts/openebs) for a full list of customizable options and use other flavors of OpenEBS Data Engines by setting the correct helm values. 
 
-Install OpenEBS helm chart with default values. 
+Install the OpenEBS helm chart with default values. 
 
 ```
 helm install openebs --namespace openebs openebs/openebs --create-namespace
 ```
 
-The above commands will install OpenEBS LocalPV Hostpath, OpenEBS LocalPV LVM, OpenEBS LocalPV ZFS, and OpenEBS Replicated Storage components in `openebs` namespace and chart name as `openebs`.
+The above commands will install OpenEBS Local PV Hostpath, OpenEBS Local PV LVM, OpenEBS Local PV ZFS, and OpenEBS Replicated Storage components in `openebs` namespace and chart name as `openebs`.
```

Comment:
LocalPV is usually one word

Reply:
We have seen in many other docs and even in our earlier docs, that the term Local PV is documented as two words. Hence, this change was done. :)